pkg/cmd/release-payload-controller/cmd.go

```diff
@@ -0,0 +1,147 @@
+	releasepayloadinformers "github.com/openshift/release-controller/pkg/client/informers/externalversions"
+	"github.com/openshift/release-controller/pkg/version"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+	prowjobclientset "k8s.io/test-infra/prow/client/clientset/versioned"
+	prowjobinformers "k8s.io/test-infra/prow/client/informers/externalversions"
+	"time"
+)
+
+type Options struct {
+	controllerContext *controllercmd.ControllerContext
+	prowNamespace     string
+	releaseNamespace  string
+}
+
+func NewReleasePayloadControllerCommand(name string) *cobra.Command {
+	o := &Options{
+		prowNamespace:    "ci",
+		releaseNamespace: "ocp",
+	}
+
+	ccc := controllercmd.NewControllerCommandConfig("release-payload-controller", version.Get(), func(ctx context.Context, controllerContext *controllercmd.ControllerContext) error {
+		o.controllerContext = controllerContext
+
+		err := o.Validate(ctx)
+		if err != nil {
+			return err
+		}
+
+		err = o.Complete(ctx)
+		if err != nil {
+			return err
+		}
+
+		err = o.Run(ctx)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+
+	cmd := ccc.NewCommandWithContext(context.Background())
+	cmd.Use = name
+	cmd.Short = "Start the release payload controller"
+
+	o.AddFlags(cmd.Flags())
+
+	return cmd
+}
+
+func (o *Options) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&o.releaseNamespace, "release-namespace", o.releaseNamespace, "The namespace where ReleasePayloads are stored.")
+	fs.StringVar(&o.prowNamespace, "prow-namespace", o.prowNamespace, "The namespace where Prow is configured to run.")
+}
+
+func (o *Options) Validate(ctx context.Context) error {
+	return nil
+}
+
+func (o *Options) Complete(ctx context.Context) error {
+	return nil
+}
+
+func (o *Options) Run(ctx context.Context) error {
+	kubeClient, err := kubernetes.NewForConfig(o.controllerContext.KubeConfig)
+	if err != nil {
+		return fmt.Errorf("can't build kubernetes client: %w", err)
+	}
+
+	// ReleasePayload Informers
+	releasePayloadClientSet, err := releasepayloadclient.NewForConfig(o.controllerContext.KubeConfig)
+	if err != nil {
+		klog.Fatalf("Error building releasePayload clientset: %s", err.Error())
+	}
+
+	releasePayloadInformerFactory := releasepayloadinformers.NewSharedInformerFactory(releasePayloadClientSet, 5*time.Minute)
+
+	// ProwJob Informers
+	prowJobClientSet, err := prowjobclientset.NewForConfig(o.controllerContext.KubeConfig)
+	if err != nil {
+		klog.Fatalf("Error building prowjob clientset: %s", err.Error())
+	}
+
+	prowJobInformerFactory := prowjobinformers.NewSharedInformerFactory(prowJobClientSet, 5*time.Minute)
+
+	// Status Controller
+	statusController, err := NewStatusController(kubeClient, o.releaseNamespace, releasePayloadInformerFactory.Release().V1alpha1().ReleasePayloads(), releasePayloadClientSet.ReleaseV1alpha1(), o.controllerContext.EventRecorder)
```

Comment:
we normally specify clients at the `releasePayloadClientSet` granularity. no need to specify .ReleaseV1alpha1